ecommerce/views/v0/__init__.py

```diff
@@ -363,6 +355,9 @@ def cart(self, request):
         """
         Returns the current cart, with the product info embedded.
         """
+        if not features.is_enabled(features.ENABLE_UPGRADE_DIALOG):
+            return Response("", status=status.HTTP_404_NOT_FOUND)
+
```

Comment:
I think we want the cart functionality to stay put - otherwise we have to enable all of the ecommerce stuff (including the upsell cards) to be able to smoke test it.